src/script/page/MainContent/panels/preferences/accountPreferences/AccountInput.tsx

```diff
@@ -112,7 +112,6 @@ const AccountInput: React.FC<AccountInputProps> = ({
   };
 
   const iconUiePrefix = rest['data-uie-name'] ?? 'account-input';
-  const fieldUieName = `${iconUiePrefix}-display`;
```

Comment:
So actually this should stay rather similar than what is was before. 
We should generate a name that has this format
`enter-<fieldname>` with `<fieldname>` being (`display-name` | `email` | `username`)

You might need to change the places where the component is instantiated